### ISSUE TITLE
"An fetal error has be found" when close the meter

### ISSUE THREAD

**Issue by terarian** [neowutran/ShinraMeter#312]
Hi,

I beilieve I got this issue pretty recently.

Basically, my Windows7 is KR version, and I play NA tera. So I have to change the language to "NA", and the UI lang to "en-US" in order for the meter to work correctly in NA tera.

 After start my computer, the first meter running works perfectly as it used to be. However, if I close the meter and run it again, the UI language will be changed to my default languae (Korean) even though the windows.xml file still has "en-US" for UI language. And when I closed the meter, I get "An fetal error has be found, please see the error.log file for more information". This issue keeps happening for the after the 1st meter (that I run after booting my computer) is closed. And the only solution for now is that I have to restart my computer, and then the 1st meter will be working perfectly again.

Could you help me with this issue?

**Comment by Gl0** [comment]
Which error is in error.log?
Probably config file stays locked after error (look at process list, may be meter is still there after error) and can't be opened by new instance of meter => It use default settings.

btw, no need to set language to "NA" when playing on NA server, "Auto" will give same result.

**Comment by terarian** [comment]
Thanks for the comment. 

1. There is nothing in the erro.log file. It's still empty after getting the error when I closed the meter.

2. You are exactly right. the meter still exist in the pricess list. And when I forced to shut it down, there will not be the same problem. But how can I fix this issue permanently? Apprently I have to force-shut down it on process list everytime I want to restart my meter.

**Comment by Gl0** [comment]
There are some issues in meter shutdown logic, we need to check it out.

**Comment by Gl0** [comment]
Check with 1.87, bug fixed or still crashing on close?

**Comment by terarian** [comment]
sorry for the late reply. I tested the lastest version, V1.90.939b018, and still got the same issue. Well, I did not get any error msg this time, but the meter is still in my process list, and the UI language of the 2nd meter (after closing the 1st raid) still went back to default.

And even thought I do not get any error message when I close 2nd or later meters, they are accumulated in the process list when I repeat close > run the meter.
